lib/rsolr/error.rb

```diff
@@ -24,6 +26,18 @@ def to_s
 
     def parse_solr_error_response body
       begin
+        # Default JSON response, try to parse and retrieve error message
+        if response[:headers] && response[:headers]["content-type"].start_with?("application/json")
```

Comment:
why "start_with?">   verus equals ignore type or something else?

Reply:
okay, I see from the test that maybe it would have "charset=utf-8" so thats why we have the `start_with?` ?

Reply:
Yes, actual content-type headers can look like `application/json; charset=utf-8`.  Perhaps they always look like that, and I should have matched that compete string?  I went with starts with `application/json` instead. Possibly that wasn't quite right.